app/editor/src/features/admin/reports/ReportSection.tsx

```diff
@@ -261,6 +261,22 @@ export const ReportSection: React.FC<IReportSectionProps> = ({ className, index
                 {folder?.description && <p>{folder?.description}</p>}
               </Col>
             </Row>
+            <Row>
+              <Col flex="1" className="description">
+                <FormikSelect
+                  label="Group Section Content By"
+                  name={`settings.${index}.settings.groupBy`}
```

Comment:
This doesn't look correct `settings.${index}.settings.groupBy`.  Did you mean `sections...`?

Reply:
you are correct